source/clear-linux/get-started/virtual-machine-install/vmware-player-2.rst

```diff
@@ -0,0 +1,469 @@
+.. _vmware-player:
+
+Use VMware\* Workstation Player
+###############################
+
+This section explains how to run |CLOSIA| as a virtual machine using the 
+`VMware Workstation 14 Player`_ hypervisor.
+
+There are 2 ways to create a |CL| VM to run in VMware:
+
+* `Method #1`: Fresh installation of |CL| into new VM.  This provides flexibility 
+  in configuring the VM size, partitions, initial |CL| bundles selection, etc.
+* `Method #2`: Use a ready-made VMware |CL| image with minimally required bundles.  
+
+Both are discussed below.  
+
+.. note::
+
+  The figures shown throughout this document are from the Windows version of 
+  VMware Workstation 14 Player.  They are applicable for the Linux version as
+  well (with minor differences).
+
+
+Install the VMware Workstation Player hypervisor
+================================================
+
+#.  Enable `Intel® Virtualization Technology
```

Comment:
Have all these links done using inline links with the full URLs at the bottom  of the page. 
 Example on an \`inline link`_  with the full link at the bottom of the page:

.. _inline link: full-url-to-be-hidden